packages/edit-post/src/hooks/components/media-upload/index.js

```diff
@@ -78,6 +78,7 @@ class MediaUpload extends Component {
 		allowedTypes,
 		multiple = false,
 		gallery = false,
+		galleryOpenInLibrary = false,
```

Comment:
I'm not convinced the prop name conveys the right meaning. I think renaming it to something like 'addToGallery' could be clearer.